tools/indexers/helpers.js

```diff
@@ -39,6 +40,11 @@ function indexModel(model, index, indexType, _queryParams, fetchOptions, entitie
         .then((entities) => _bulkIndexEntities(entities, index, indexType))
         // eslint-disable-next-line no-loop-func
         .then((resp) => {
+          if (resp && resp.errors) {
+            const failedModels = _.filter(resp.items, 'index.error');
+            console.error(`${failedModels.length} failed to reindex.`);
+            throw _.map(failedModels, (item) => item.index);
+          }
```

Comment:
According to the docs, `client.bulk()`, which is what's being returned by `_bulkIndexEntities()`, returns a promise that is rejected on errors. So, if an error occurred, the `.then()` block would never be executed, so this `if (resp && resp.errors)` block does nothing.

What am I missing?